tools/accuracy_checker/accuracy_checker/annotation_converters/ms_coco.py

```diff
@@ -257,11 +271,55 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
 
         return ConverterReturn(keypoints_annotations, {'label_map': label_map}, None)
 
+def make_segmentation_mask(height, width, path_to_mask, labels, segmentations):
+    polygons = []
+    for mask in segmentations:
+        rles = maskUtils.frPyObjects(mask, height, width)
+        rle = maskUtils.merge(rles)
+        polygons.append(rle)
+
+    masks = []
+    for polygon in polygons:
+        mask = maskUtils.decode(polygon)
+        if len(mask.shape) < 3:
+            mask = np.expand_dims(mask, axis=-1)
+        masks.append(mask)
+    if masks:
+        masks = np.stack(masks, axis=-1)
+    else:
+        masks = np.zeros((height, width, 0), dtype=np.uint8)
+    masks = (masks * np.asarray(labels, dtype=np.uint8)).max(axis=-1)
+    mask = np.squeeze(masks)
+
+    image = Image.frombytes('L', (width, height), mask.tostring())
+    image.save(path_to_mask)
+
+    return mask
+
 
 class MSCocoSegmentationConverter(MSCocoDetectionConverter):
     __provider__ = 'mscoco_segmentation'
     annotation_types = (CoCoInstanceSegmentationAnnotation, )
 
+    @classmethod
+    def parameters(cls):
+        configuration_parameters = super().parameters()
+        configuration_parameters.update({
+            'semantic_only': BoolField(
+                optional=True, default=False, description="Semantic segmentation only mode."
+            ),
+            'masks_dir': PathField(
+                is_directory=True, optional=True,
+                description='path to segmentation masks, used if semantic_only is True'
+            ),
+        })
+        return configuration_parameters
+
+    def configure(self):
+        super().configure()
+        self.semantic_only = self.get_value_from_config('semantic_only')
+        self.masks_dir = self.get_value_from_config('masks_dir')
```

Comment:
we need check that mask_dir provided if semantic_only enabled and raise error or define default path for saving (self.annotation_file.parent/'masks').  Also we need create dir if it does not exists in this case.